### ISSUE TITLE
As a DEVELOPER I want THE CREATE CHEEP METHODS TO FOLLOW THE ONION ARCHITECTURE

### ISSUE THREAD

**Issue by Thyme-Cat** [ITU-BDSA2024-GROUP30/Chirp#155]
**Acceptance Criteria:** 
- In `CreatingCheeps.cshtml.cs` the `OnPostAsync()` should call `CheepService.cs` instead of `CheepRepository.cs`
- `CheepService.cs` should then call `CheepRepository.cs` with the necessary information to create and save a cheep there.
- `CreatingCheeps.cshtml.cs` should not be creating the cheep object directly, and instead give it to `CheepService.cs` which gives it to `CheepRepository.cs`
- Bottom line, it should follow the onion architecture that we already have.

**Comment by tomat-suppe** [comment]
We revisited this and realised the issue wasn't needed.
